### DISCUSSION TITLE
gives error on installing argon2

### DISCUSSION THREAD

**Discussion by vikramacn5** [coder/code-server#4408] [Q&A] [answered]
yarn global add code-server

The above command in termux gives an error saying argon2 is not found how to fix it.

![image](https://user-images.githubusercontent.com/57588103/139179665-eac772cc-d96c-4444-8656-2bbc6676f506.png)

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
yeah yeah yeah the argon2 dependency has errors. I am experiencing it as well as more people with Raspberry Pis, and x86-based systems(Android, Raspberry Pis). Tracked in #4154

**Reply by vikramacn5** [reply]
you can try download release for arm64,install nodejs/yarn in termux,and delete node_modules directory,delete /lib/node,create /lib/node input node $@ ,exec yarn install on project root dir,

I found this somewhere else but I didn't understand, do you have any idea.

**Reply by lg-nineoneninethreenine** [reply]
Actually, I think that will fail.

**Reply by jsjoeio** [reply]
Sorry about the issues but @im-coder-lg is right. Hoping we can fix this somehow.
